src/Templates/DiffPlaces/createAllBadOpcodeTests.sh

```diff
@@ -28,18 +26,17 @@
 ./createBadOpcodeTest.sh 2D
 ./createBadOpcodeTest.sh 2E
 ./createBadOpcodeTest.sh 2F
-./createBadOpcodeTest.sh 49
-./createBadOpcodeTest.sh 4A
+./createBadOpcodeTest.sh 49 '<Cancun'
```

Comment:
can it accept list?  `Berlin, London, Merge, Shanghai` ?

Reply:
Not yet. I can add that, when I have time - but I can't promise it'll be this week. Sorry, we are revamping our docs site.